app/src/main/java/com/github/godspeed010/weblib/FolderColorGroup.kt

```diff
@@ -0,0 +1,60 @@
+package com.github.godspeed010.weblib
```

Comment:
Honestly, I liked the new implementation somewhat better simply because to change the folder color or retrieve the current selected color, it was just a single variable for either. Some of the thought was that if the color system is changed later on, it'd be easier to change it in a separate file than find the function for it in the LibraryFragment code.

Reply:
I have changed it so it uses your code, also I apologize for cases where I use Kotlin inefficiently, I had not touched Kotlin nor Android development in depth prior to making the changes necessary for the Folders to have colors.

Reply:
No worries, the code looks great, and I'm still learning more all the time too! Thanks for making the changes